test/integration/run-command/run_command.spec.test.ts

```diff
@@ -2,5 +2,13 @@ import { loadSpecTests } from '../../spec';
 import { runUnifiedSuite } from '../../tools/unified-spec-runner/runner';
 
 describe('RunCommand spec', () => {
-  runUnifiedSuite(loadSpecTests('run-command'));
+  runUnifiedSuite(loadSpecTests('run-command'), test => {
+    if (test.description.includes('timeoutMS') || test.description.includes('timeoutMode')) {
```

Comment:
Are there still some tests left for this in run-command? I thought they were all moved to the CSOT spec folder. If there are some of these tests left, we should reference a TODO(NODE-xxxx) here where we would do the part of CSOT implementation that would unskip these.

For the tests that are in the CSOT spec folder, we should still sync these new tests now, and create the placeholder to run & skip CSOT spec tests with the same `TODO` reference because otherwise we might miss them when going to implement CSOT (we could add a callout to the corresponding JIRA to make sure to include them, but I think just syncing them is more straightforward and less hoops to jump).

Reply:
Should we sync all of CSOT now? I'm just thinking we're going to need to sync all of CSOT, and the state the tests are in now may not be what they are when it is time to do that, so having a snapshot or portion of the tests might be confusing later.

This is going to be the case for all drivers other than go and python that can sync and run the CSOT tests now.

Reply:
the tests you introduced are in their own files, so I think it's fine not to sync the others; when we go to work on CSOT, there will be tickets referencing specific commits which we can sync and then update any corresponding tests accordingly; as for the other drivers - we should call out in downstream changes or the ticket description that drivers should ensure that they add these new tests now or in their own CSOT follow up tickets